packages/gatsby-link/tsconfig.json

```diff
@@ -1,3 +1,4 @@
 {
-  "extends": "../../tsconfig.json"
+  "extends": "../../tsconfig.json",
+  "exclude": ["src/__tests__", "dist/"]
```

Comment:
why is this required? I don't think the other projects needed this exclude.

Reply:
I saw this pattern in `packages/gatsby-core-utils/tsconfig.json`

Basically if there's a "typegen" step in NPM, tsc will generate .d.ts files for tests. This causes two problems:

1. Looks like jest tries to run the output ".d.ts" files as tests and complains there are no test cases
2. We wouldn't want __tests__ to be exportable types.